cmd/docker-app/main.go

```diff
@@ -22,6 +22,7 @@ func main() {
 			}
 			return nil
 		}
+		_ = cmd.Flags().MarkHidden("")
```

Comment:
What is this line for? 🤔

Reply:
just some crappy experiment I committed by mistake